torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -2267,21 +2267,24 @@ void InitXlaModuleBindings(py::module m) {
       .def("_get_stablehlo",
            [](const std::vector<at::Tensor>& tensors, const std::string& device,
               const std::vector<std::string>& devices,
-              bool emit_bytecode) -> py::bytes {
-            NoGilSection nogil;
-            EmitMode mode = emit_bytecode ? EmitMode::kStableHloBytecode
-                                          : EmitMode::kStableHloReadable;
-            std::vector<XLATensorPtr> xtensors;
-            if (tensors.empty()) {
-              torch::lazy::BackendDevice backend_device =
-                  GetDeviceOrCurrent(device);
-              xtensors =
-                  XLAGraphExecutor::Get()->GetLiveTensors(&backend_device);
-            } else {
-              xtensors = GetXlaTensors(tensors, /*want_all=*/false);
+              bool emit_bytecode) {
```

Comment:
I don't understand why we should remove the `py::bytes` return type. The PR description says it's undefined behavior - how?

Reply:
likely this change doesn't matter. C++ infers declared return type for lambdas

Reply:
Added back the py::bytes return type.